src/manage.c

```diff
@@ -298,6 +299,12 @@ static acq_status_t * choose_acq_sat(void)
   gps_time_t t = get_current_time();
 
   for (u32 i=0; i<PLATFORM_SIGNAL_COUNT; i++) {
+    if (CODE_GPS_L2CM == acq_status[i].sid.code) {
```

Comment:
For consistency I'd also make the same check in the second loop later in this function